INSTALL.md

```diff
@@ -9,23 +9,31 @@ To install subscription-manager, run
 sudo dnf install subscription-manager
 ```
 
+This is not necessary on RHEL installations as subscription-manager is pre-installed and registration is required to install any packages with DNF.
+
 ## Developer installation
 
-The process below has been tested on Fedora 36.
-Other Fedora versions or distributions may require some adaptation.
+The process below has been tested on Fedora 36 and RHEL 9.
+Other Fedora/RHEL versions or distributions may require some adaptation.
 
 1. First you need to install RPM packages to build and run subscription-manager binaries:
 
    ```bash
    sudo dnf install git gcc python3-devel openssl-devel glib2-devel \
        python3-rpm python3-librepo libdnf-devel cmake
    ```
+   
+   On RHEL, you need to enable the [CodeReady Linux Builder](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/package_manifest/repositories#CodeReadyLinuxBuilder-repository) repository in order to gain access to the development packages like `libdnf-devel`.
 
+   ```bash
+   sudo subscription-manager repos --enable codeready-builder-for-rhel-9-x86_64-rpms
+   ```
+   
    <!-- libdnf-devel, cmake are required to build product-id plugin -->
```

Comment:
Please leave this comment directly under the `sudo dnf install` block.

Reply:
Makes sense! Moved.